docs/products/kafka/kafka-connect/howto/jdbc-source-connector-pg.rst

```diff
@@ -0,0 +1,113 @@
+
+.. Note::
+
+    If you're using Aiven for PostgreSQL the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a PostgreSQL JDBC source connector with Aiven CLI
+-------------------------------------------------------
+
+The following example demonstrates how to setup an Apache Kafka JDBC source connector to a PostgreSQL database using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`:
+
+1. In a file named ``jdbc_source_pg.json`` define the connector configuration with the following content:
+
+::
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class":"io.aiven.connect.jdbc.JdbcSourceConnector",
+        "connection.url":"jdbc:postgresql://PG_HOST:PG_PORT/PG_DATABASE_NAME?sslmode=SSL_MODE",
+        "connection.user":"PG_USER",
+        "connection.password":"PG_PASSWORD",
+        "table.whitelist":"PG_TABLES",
+        "mode":"JDBC_MODE",
+        "topic.prefix":"KAFKA_TOPIC_PREFIX",
+        "tasks.max":"NR_TASKS",
+        "poll.interval.ms":"POLL_INTERVAL"
+    }
+
+.. Note::
+
+    The ``PG_HOST``, ``PG_PORT``, ``PG_DATABASE_NAME``, ``SSL_MODE``, ``PG_USER``, ``PG_PASSWORD`` and ``PG_TABLES`` are the parameters collected in the :ref:`prerequisite <connect_jdbc_pg_source_prereq>` phase. 
+    The following additional parameters need to be set:
+
+    * ``name``: the connector name
+    * ``mode``: the query mode, more information in the :doc:`dedicated page <../concepts/jdbc-source-modes>`; depending on the selected mode, additional configuration entries might be required.
+    * ``topic.prefix``: the prefix that will be used for topic names; the resulting topic name is the concatenation of the ``topic.prefix`` and the table name.
+    * ``tasks.max``: maximum number of tasks to execute in parallel
+    * ``poll.interval.ms``: query frequency
+
+    Check out the `dedicated documentation <https://github.com/aiven/jdbc-connector-for-apache-kafka/blob/master/docs/source-connector-config-options.rst>`_ for the full list of parameters.
+
+2. Execute the following :ref:`Aiven CLI command <avn_service_connector_create>` to create the connector, replacing the ``SERVICE_NAME`` with the name of the Aiven service where the connector needs to run:
+
+:: 
+
+    avn service connector create SERVICE_NAME @jdbc_source_pg.json
+
+3. Check the connector status with the following command, replacing the ``SERVICE_NAME`` with the Aiven service and the ``CONNECTOR_NAME`` with the name of the connector defined before:
+
+::
+
+    avn service connector status SERVICE_NAME CONNECTOR_NAME
+
+4. Verify in the Aiven for Apache Kafka target instance, the presence of the topic and the data
+
+Example: define a JDBC incremental connector
```

Comment:
Example helped me a lot, I tried to run a command without looking at it, but was getting errors.

Reply:
:)